src/seedu/addressbook/commands/EditCommand.java

```diff
@@ -0,0 +1,47 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList.PersonNotFoundException;
+
+/**
+ * Edits a person's name identified using it's last displayed index from the address book.
+ */
+
+public class EditCommand extends Command {
+    public static final String COMMAND_WORD = "edit";
```

Comment:
Just minor formatting issue; might want to follow how in other command files, there is no empty line between the header comment and method declaration and there is an empty line between the method declaration and declaration of the first variable.